crates/ruff_linter/src/rules/pyupgrade/rules/redundant_open_modes.rs

```diff
@@ -72,30 +72,7 @@ pub(crate) fn redundant_open_modes(checker: &mut Checker, call: &ast::ExprCall)
     }
 
     match call.arguments.find_argument("mode", 1) {
```

Comment:
Should I rewrite this block using `let-else`?

Reply:
Yes please, I think that would be cleaner :-)